src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
```

Comment:
There is an unnecessary space between `.stars--1` and `:nth-child(1)`. This could potentially lead to specificity issues or unintended styling. It should be `.stars--1:nth-child(1)`. Please check and correct similar issues in other selectors.